renovate.json

```diff
@@ -1,6 +1,11 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:recommended", ":semanticCommitScope(package)"],
+  "extends": [
+    "config:recommended",
+    ":semanticCommitScope(package)",
+    "github>marinatedconcrete/config//renovate/marinatedconcrete#renovate-config-2.0.0",
+    "github>marinatedconcrete/config//renovate/devcontainer#renovate-config-2.0.0"
```

Comment:
Should this be pinned? We probably want to use the version that's currently in the repo

Reply:
Annoyingly, renovate doesn't let you include local files at run time; it'll do a fetch regardless.  That means the default branch, which is /probably/ fine for us, but I went with this for two reasons:
1) eating our dogfood, so-to-speak.  This is how others should also use this, so it services as an example
2) it adds a bit more determinism, IMO